app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/event_target_parser.rb

```diff
@@ -20,6 +39,9 @@ def parse
         :manager_id  => ems_event.ext_management_system.id,
         :event_id    => ems_event.id
       )
+    when /^volume/
+      storage_manager = ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager.find_by(:parent_ems_id => ems_event[:ems_id])
+      targets << storage_manager
```

Comment:
Similar here, you can do `targets << ems_event.ext_management_system.storage_manager`